src/kv/value.rs

```diff
@@ -514,14 +564,12 @@ macro_rules! impl_value_to_primitive {
     }
 }
 
-impl_to_value_primitive![
-    usize, u8, u16, u32, u64, u128, isize, i8, i16, i32, i64, i128, f32, f64, char, bool,
-];
+impl_to_value_primitive![usize, u8, u16, u32, u64, isize, i8, i16, i32, i64, f32, f64, char, bool,];
```

Comment:
Nit: final `,`.

Reply:
I think that macro implementation is a little iffy and it actually requires that trailing `,` 😄